app/views/school/welcome_wizard/la_funded_devices_you_can_order.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-grid-row">
```

Comment:
this page will be missing a title see https://github.com/DFE-Digital/get-help-with-tech/blob/main/app/views/school/welcome_wizard/allocation.html.erb#L1